src/shared/features/AppletSettings/ExportDataSetting/ExportDataSetting.tsx

```diff
@@ -97,71 +101,92 @@ export const ExportDataSetting = () => {
   };
 
   return (
-    <FormProvider {...methods}>
-      <form noValidate autoComplete="off">
-        <StyledAppletSettingsDescription>{t('exportDescription')}</StyledAppletSettingsDescription>
-        <SelectController
-          name={'dateType'}
-          control={control}
-          options={getDateTypeOptions()}
-          label={t('dateRange')}
-          data-testid={`${dataTestid}-dateType`}
-          dropdownStyles={{
-            width: '15rem',
-          }}
-          SelectProps={{
-            autoWidth: true,
-          }}
-          customChange={onDateTypeChange}
+    <>
+      <Modal
```

Comment:
Proposing to move this modal to a separate component within the ExportDataSetting directory, what do you think?

Reply:
Sounds good!